src/index.ts

```diff
@@ -1,7 +1,20 @@
 const chars = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ_$';
 const reserved = /^(?:do|if|in|for|int|let|new|try|var|byte|case|char|else|enum|goto|long|this|void|with|await|break|catch|class|const|final|float|short|super|throw|while|yield|delete|double|export|import|native|return|switch|throws|typeof|boolean|default|extends|finally|package|private|abstract|continue|debugger|function|volatile|interface|protected|transient|implements|instanceof|synchronized)$/;
 const unsafe = /[<>\/\u2028\u2029]/g;
-const escaped: Record<string, string> = { '<': '\\u003C', '>' : '\\u003E', '/': '\\u002F', '\u2028': '\\u2028', '\u2029': '\\u2029' };
+const escaped: Record<string, string> = {
+	'<': '\\u003C',
+	'>' : '\\u003E',
+	'/': '\\u002F',
+	'\\': '\\\\',
+	'\b': '\\b',
+	'\f': '\\f',
+	'\n': '\\n',
+	'\r': '\\r',
+	'\t': '\\t',
+	'\0': '\\u0000',
```

Comment:
Why not just escape this as `\0`? (It's _not_ an octal escape in JS.)

Reply:
Is the goal to match `JSON.stringify`?

Reply:
It's not an explicit goal, I just didn't realise it wasn't necessary